lldb/source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.cpp

```diff
@@ -864,7 +868,9 @@ swift::FuncDecl *SwiftASTManipulator::GetFunctionToInjectVariableInto(
   // When not binding generic type parameters, we want to inject the metadata
   // pointers in the wrapper, so we can pass them as opaque pointers in the
   // trampoline function later on.
-  if (m_bind_generic_types == lldb::eDontBind && variable.IsMetadataPointer())
+  if (m_bind_generic_types == lldb::eDontBind &&
```

Comment:
Are we relying in the same `--bind-generic-types` mechanism to enable this behavior? It seems to me this would never work in the "bind" mode.